src/index.js

```diff
@@ -413,6 +413,20 @@ class Masq {
     let db = this._getDB()
     return db.delAsync(key)
   }
+
+  async list (prefix) {
+    let db = this._getDB()
+    const list = await db.listAsync(prefix)
+    const reformattedList = list.map(e => {
+      const el = Array.isArray(e) ? e[0] : e
+
+      return {
```

Comment:
As we said, we will change this to return an dict like `{ key1: value, key2: value2 }`